src/NUnitFramework/framework/Internal/Reflect.cs

```diff
@@ -226,5 +226,90 @@ public static object InvokeMethod( MethodInfo method, object fixture, params obj
         }
 
         #endregion
+
+#if NETSTANDARD1_3
+        /// <summary>
+        /// <para>
+        /// Selects the ultimate shadowing property just like <see langword="dynamic"/> would,
+        /// rather than throwing <see cref="AmbiguousMatchException"/>
+        /// for properties that shadow properties of a different property type.
+        /// </para>
+        /// <para>
+        /// If you request both public and nonpublic properties, every public property is preferred
+        /// over every nonpublic property. It would violate the principle of least surprise for a
+        /// derived class�s implementation detail to be chosen over the public API for a type.
```

Comment:
There's a corrupted character in this line - repeated for the other platforms as well. 🙂

Reply:
Oh yeah, I forgot to save as UTF8 when I moved this function into this file.
That's a terrific eye you have, thank you!